packages/styles/src/calendar.scss

```diff
@@ -340,16 +205,102 @@ $fd-calendar-special-days: (
     }
   }
 
+  &__item {
+    @include fd-reset();
+
+    overflow: hidden;
+    position: relative;
+    height: var(--fdCalendar_Item_Height);
+    margin: var(--fdCalendar_Item_Spacing);
+    min-width: var(--fdCalendar_Item_Width);
+    border-radius: var(--fdCalendar_Border_Radius);
+    background-color: var(--fdCalendar_Item_Background);
+    flex: 1 1 calc(var(--fdCalendar_Item_Current_Border_Width) * 2 + var(--fdCalendar_Item_Spacing) * 2);
+    cursor: pointer;
+    display: inline-block;
+    border: var(--fdCalendar_Item_Border);
+
+    @include fd-hover() {
+      background-color: var(--sapList_Hover_Background);
```

Comment:
you can use a CSS variable for the hover background and then instead of every time writing
```
@include fd-hover() {
    --fdCalendar_Item_Background: var(--sapList_Hover_Background);
}
```

you can just change the var value. 
For example:
`--fdCalendar_Item_Background_Hover: var(--sapList_Hover_Background);`

Here on line 224 you do:
```
@include fd-hover() {
    background-color: var(--fdCalendar_Item_Background_Hover);
}
```

and then on lines 246, 266, etc you just do:
```
--fdCalendar_Item_Background: new value
```